mona_sdk/client.py

```diff
@@ -64,6 +64,10 @@
 
 SHOULD_USE_SSL = get_boolean_value_for_env_var("MONA_SDK_SHOULD_USE_SSL", True)
 
+APP_SERVER_URL = os.environ.get("MONA_SDK_APP_SERVER_URL")
```

Comment:
please rename to OVERRIDE_MONA_SDK_APP_SERVER_URL (both global const name + env var name)